deploy/ods-pipeline/Chart.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+name: ods-pipeline
+description: Umbrella chart for ods-pipeline
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.2.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.2.0"
+
+dependencies:
+  - name: images
+    version: 0.2.0
+    condition: images.enabled
+  - name: setup
```

Comment:
Did you consider some alternative names? I cannot provide a better name right now, but `setup` feels a bit odd to me. `environment` comes to mind, but isn't better really.

A somewhat related thought: do we envision use cases where one would NOT install the `tasks` sub chart? It feels to me like `setup` and `tasks` are always going to be installed, and `images` is optional. Just wondering what your thoughts are. In case you also think both will always be installed, we could think about merging them, and potentially then we have a better name? Or even moving setup/task into the main chart, and only having images as a subchart?

Anyway, just food for thought ... I'm happy to go forward with the current state as well.

Reply:
Good point, `setup` is probably not a good choice, @henninggross and I were brainstorming a bit but couldn't come up with a good name so far. Shall we go with this one for the moment until we find a better name? 

With respect to the three subcharts: One benefit of keeping them separate is that one can basically still set up `ods-pipeline` with central `ClusterTask`, `BuildConfig` and `ImageStream` resources without too much effort -- just in case somebody favors this more restrictive governance approach.